cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/BufferingTable.java

```diff
@@ -395,6 +397,12 @@ public Row get(byte[] row, byte[] startColumn, byte[] stopColumn, int limit) {
    */
   @Override
   public void put(byte[] row, byte[][] columns, byte[][] values) {
+    putInternal(row, columns, values);
+    // report metrics _after_ write was performed
+    reportWrite(1, getSize(row) + getSize(columns) + getSize(values));
```

Comment:
yes. We report here ops of the dataset.. We should also report datastore ops - in addition.